include/rcpputils/time.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCPPUTILS__TIME_HPP_
+#define RCPPUTILS__TIME_HPP_
+
+#include <chrono>
+
+#include "rcutils/time.h"
+
+#include "rcpputils/visibility_control.hpp"
+
+namespace rcpputils
+{
+
+/// Convert to rcutils duration value.
+/*
+ * \param[in] time The time to be converted to rcutils duration.
+ * \return rcutils duration value
+ * \throws std::invalid_argument if time is bigger than std::chrono::nanoseconds::max().
+ */
+template<typename DurationRepT = int64_t, typename DurationT = std::milli>
+RCPPUTILS_PUBLIC
+rcutils_duration_value_t convert_to_rcutils_duration(
```

Comment:
how about returning as `std::chrono::nanoseconds` instead of `rcutils_duration_value_t` as C++ context?

Reply:
Yeah. But if use `std::chrono::nanoseconds`, user needs to call count().  

How about this ?
```c++
template<typename DurationRepT = int64_t, typename DurationT = std::milli>
RCPPUTILS_PUBLIC
int64_t convert_to_nanoseconds(const std::chrono::duration<DurationRepT, DurationT> & time)
```

Reply:
> But if use std::chrono::nanoseconds, user needs to call count().

then user can call `count` method? imo, i think that is okay for user application.

Reply:
Okay.